v3/bind.go

```diff
@@ -6,6 +6,7 @@ import (
 	enchex "encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/Azure/go-ntlmssp"
```

Comment:
nitpick - I prefer to see third party imports grouped separately from stdlib imports.  You can move this to right after the `ber` import.